docs/runtime-contract.md

```diff
@@ -279,7 +279,7 @@ code. These settings apply to both `livenessProbe` and `readinessProbe`:
 - `periodSeconds` set to platform-specific value
 
 In order to enable scaling in response to load, developers SHOULD NOT set the
```

Comment:
"Developers SHOULD NOT" should be changed to a non-keyword. We allow setting initialDelaySeconds greater than zero and developers might do this.

"SHOULD aim to minimize" should be changed to a non-keyword.

Reply:
see what you think of the new wording